hummingbot/client/command/balance_command.py

```diff
@@ -84,7 +86,14 @@ def balance(self,
     async def show_balances(self):
         total_col_name = f'Total ({RateOracle.global_token_symbol})'
         self._notify("Updating balances, please wait...")
-        all_ex_bals = await UserBalances.instance().all_balances_all_exchanges()
+        network_timeout = float(global_config_map["other_commands_timeout"].value)
+        try:
+            all_ex_bals = await asyncio.wait_for(
+                UserBalances.instance().all_balances_all_exchanges(), network_timeout
+            )
+        except asyncio.TimeoutError:
+            self._notify("\nA network error prevented the balances to update. See logs for more details.")
+            raise
```

Comment:
I don't believe we should raise an error here if we're not catching it.

Reply:
My thinking for raising was that the traceback is logged and we can know where things went wrong. For instance, here, a number of different exchanges can get throttled. With the traceback, we'll know which one.